test/seed.js

```diff
@@ -693,15 +743,39 @@ describe("Contract: Seed", async () => {
               permissionedSeed,
               fee
           );
+          await setup.data.seed
+              .connect(admin)
+              .addClass(hardCap, CLASS_PERSONAL_FUNDING_LIMIT, price, 10000000, 1700000000);
 
           await setup.data.seed
               .connect(admin)
-              .addClass(1e14, 1e12, 1e12, 10000000);
+              .addClass(1e14, 1e12, 1e12, 10000000, 1700000000);
```

Comment:
Can you pls transform all numeric values into constants in the beginning of the file?
It was my bad, I totally forgot to do it when I was writing tests

Reply:
Created 
  const CLASS_VESTING_DURATION =  10000000;
  const CLASS_VESTING_START_TIME = 1700000000;

Reply:
Can you do the same for 1e14 and 1e12  pls?